{"level":"info","message":"Backend rodando na porta 3000"}
{"code":"ER_BAD_DB_ERROR","errno":1049,"level":"error","message":"Erro ao conectar no banco de dados: Unknown database 'rover_db'","sqlMessage":"Unknown database 'rover_db'","sqlState":"42000","stack":"Error: Unknown database 'rover_db'\n    at Packet.asError (/app/node_modules/mysql2/lib/packets/packet.js:738:17)\n    at ClientHandshake.execute (/app/node_modules/mysql2/lib/commands/command.js:29:26)\n    at Connection.handlePacket (/app/node_modules/mysql2/lib/connection.js:481:34)\n    at PacketParser.onPacket (/app/node_modules/mysql2/lib/connection.js:97:12)\n    at PacketParser.executeStart (/app/node_modules/mysql2/lib/packet_parser.js:75:16)\n    at Socket.<anonymous> (/app/node_modules/mysql2/lib/connection.js:104:25)\n    at Socket.emit (node:events:517:28)\n    at addChunk (node:internal/streams/readable:368:12)\n    at readableAddChunk (node:internal/streams/readable:341:9)\n    at Readable.push (node:internal/streams/readable:278:10)"}
{"level":"info","message":"Backend rodando na porta 3000"}
{"code":"ER_BAD_DB_ERROR","errno":1049,"level":"error","message":"Erro ao conectar no banco de dados: Unknown database 'rover_db'","sqlMessage":"Unknown database 'rover_db'","sqlState":"42000","stack":"Error: Unknown database 'rover_db'\n    at Packet.asError (/app/node_modules/mysql2/lib/packets/packet.js:738:17)\n    at ClientHandshake.execute (/app/node_modules/mysql2/lib/commands/command.js:29:26)\n    at Connection.handlePacket (/app/node_modules/mysql2/lib/connection.js:481:34)\n    at PacketParser.onPacket (/app/node_modules/mysql2/lib/connection.js:97:12)\n    at PacketParser.executeStart (/app/node_modules/mysql2/lib/packet_parser.js:75:16)\n    at Socket.<anonymous> (/app/node_modules/mysql2/lib/connection.js:104:25)\n    at Socket.emit (node:events:517:28)\n    at addChunk (node:internal/streams/readable:368:12)\n    at readableAddChunk (node:internal/streams/readable:341:9)\n    at Readable.push (node:internal/streams/readable:278:10)"}
{"level":"info","message":"Backend rodando na porta 3000"}
{"code":"ER_BAD_DB_ERROR","errno":1049,"level":"error","message":"Erro ao conectar no banco de dados: Unknown database 'rover_db'","sqlMessage":"Unknown database 'rover_db'","sqlState":"42000","stack":"Error: Unknown database 'rover_db'\n    at Packet.asError (/app/node_modules/mysql2/lib/packets/packet.js:738:17)\n    at ClientHandshake.execute (/app/node_modules/mysql2/lib/commands/command.js:29:26)\n    at Connection.handlePacket (/app/node_modules/mysql2/lib/connection.js:481:34)\n    at PacketParser.onPacket (/app/node_modules/mysql2/lib/connection.js:97:12)\n    at PacketParser.executeStart (/app/node_modules/mysql2/lib/packet_parser.js:75:16)\n    at Socket.<anonymous> (/app/node_modules/mysql2/lib/connection.js:104:25)\n    at Socket.emit (node:events:517:28)\n    at addChunk (node:internal/streams/readable:368:12)\n    at readableAddChunk (node:internal/streams/readable:341:9)\n    at Readable.push (node:internal/streams/readable:278:10)"}
{"level":"info","message":"Backend rodando na porta 3000"}
{"address":"172.18.0.2","code":"ECONNREFUSED","errno":-111,"fatal":true,"level":"error","message":"Erro ao conectar no banco de dados: connect ECONNREFUSED 172.18.0.2:3306","port":3306,"stack":"Error: connect ECONNREFUSED 172.18.0.2:3306\n    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)","syscall":"connect"}
{"fatal":true,"level":"error","message":"Erro ao limpar tabela de coordenadas: Can't add new command when connection is in closed state","stack":"Error: Can't add new command when connection is in closed state\n    at Connection._addCommandClosedState (/app/node_modules/mysql2/lib/connection.js:164:17)\n    at Connection.query (/app/node_modules/mysql2/lib/connection.js:575:17)\n    at /app/server.js:46:6\n    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)\n    at next (/app/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/app/node_modules/express/lib/router/route.js:119:3)\n    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)\n    at /app/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/app/node_modules/express/lib/router/index.js:346:12)\n    at next (/app/node_modules/express/lib/router/index.js:280:10)"}
{"fatal":true,"level":"error","message":"Erro ao inserir no banco de dados: Can't add new command when connection is in closed state","stack":"Error: Can't add new command when connection is in closed state\n    at Connection._addCommandClosedState (/app/node_modules/mysql2/lib/connection.js:164:17)\n    at Connection.query (/app/node_modules/mysql2/lib/connection.js:575:17)\n    at /app/server.js:56:6\n    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)\n    at next (/app/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/app/node_modules/express/lib/router/route.js:119:3)\n    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)\n    at /app/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/app/node_modules/express/lib/router/index.js:346:12)\n    at next (/app/node_modules/express/lib/router/index.js:280:10)"}
{"fatal":true,"level":"error","message":"Erro ao limpar tabela de rovers: Can't add new command when connection is in closed state","stack":"Error: Can't add new command when connection is in closed state\n    at Connection._addCommandClosedState (/app/node_modules/mysql2/lib/connection.js:164:17)\n    at Connection.query (/app/node_modules/mysql2/lib/connection.js:575:17)\n    at /app/server.js:155:6\n    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)\n    at next (/app/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/app/node_modules/express/lib/router/route.js:119:3)\n    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)\n    at /app/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/app/node_modules/express/lib/router/index.js:346:12)\n    at next (/app/node_modules/express/lib/router/index.js:280:10)"}
{"fatal":true,"level":"error","message":"Erro ao limpar tabela de coordenadas: Can't add new command when connection is in closed state","stack":"Error: Can't add new command when connection is in closed state\n    at Connection._addCommandClosedState (/app/node_modules/mysql2/lib/connection.js:164:17)\n    at Connection.query (/app/node_modules/mysql2/lib/connection.js:575:17)\n    at /app/server.js:46:6\n    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)\n    at next (/app/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/app/node_modules/express/lib/router/route.js:119:3)\n    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)\n    at /app/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/app/node_modules/express/lib/router/index.js:346:12)\n    at next (/app/node_modules/express/lib/router/index.js:280:10)"}
{"fatal":true,"level":"error","message":"Erro ao inserir no banco de dados: Can't add new command when connection is in closed state","stack":"Error: Can't add new command when connection is in closed state\n    at Connection._addCommandClosedState (/app/node_modules/mysql2/lib/connection.js:164:17)\n    at Connection.query (/app/node_modules/mysql2/lib/connection.js:575:17)\n    at /app/server.js:56:6\n    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)\n    at next (/app/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/app/node_modules/express/lib/router/route.js:119:3)\n    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)\n    at /app/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/app/node_modules/express/lib/router/index.js:346:12)\n    at next (/app/node_modules/express/lib/router/index.js:280:10)"}
{"level":"info","message":"Backend rodando na porta 3000"}
{"code":"ENOTFOUND","errno":-3008,"fatal":true,"hostname":"db","level":"error","message":"Erro ao conectar no banco de dados: getaddrinfo ENOTFOUND db","stack":"Error: getaddrinfo ENOTFOUND db\n    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:107:26)","syscall":"getaddrinfo"}
{"level":"info","message":"Backend rodando na porta 3000"}
{"code":"ENOTFOUND","errno":-3008,"fatal":true,"hostname":"db","level":"error","message":"Erro ao conectar no banco de dados: getaddrinfo ENOTFOUND db","stack":"Error: getaddrinfo ENOTFOUND db\n    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:107:26)","syscall":"getaddrinfo"}
{"level":"info","message":"Backend rodando na porta 3000"}
{"level":"info","message":"Backend rodando na porta 3000"}
{"level":"info","message":"Backend rodando na porta 3000"}
{"level":"info","message":"Backend rodando na porta 3000"}
{"level":"info","message":"Backend rodando na porta 3000"}
{"level":"info","message":"Backend rodando na porta 3000"}
{"level":"info","message":"Backend rodando na porta 3000"}
{"level":"info","message":"Backend rodando na porta 3000"}
{"level":"info","message":"Backend rodando na porta 3000"}
{"level":"info","message":"Backend rodando na porta 3000"}
{"level":"info","message":"Backend rodando na porta 3000"}
{"level":"info","message":"Backend rodando na porta 3000"}
{"level":"info","message":"Backend rodando na porta 3000"}
{"level":"info","message":"Backend rodando na porta 3000"}
{"level":"info","message":"Backend rodando na porta 3000"}
{"level":"info","message":"Backend running on the port 3000"}
{"level":"info","message":"Backend running on the port 3000"}
{"level":"info","message":"Backend running on the port 3000"}
{"level":"info","message":"Backend running on the port 3000"}
{"level":"info","message":"Backend running on the port 3000"}
